src/LintingViewExtension/LintingViewExtension.cs

```diff
@@ -52,7 +57,21 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
 
         private void OnRequestOpenLinterView(object sender, System.EventArgs e)
         {
-            if (linterMenuItem.IsChecked) return;
+            if (linterMenuItem.IsChecked)
+            {
+                if (linterView != null && 
+                    (dynamoView.tabDynamic.SelectedItem as TabItem).Header.ToString() != EXTENSION_NAME)
+                {
+                    dynamoView.tabDynamic.SelectedItem = dynamoView
```

Comment:
I think it would be better to move this code into ViewLoadedParams.cs and call that API from here. In that case, we can avoid creating the private dynamoView variable.